### ISSUE TITLE
EOL warning for highlight.js when installing @nomiclabs/hardhat-truffle5

### ISSUE THREAD

**Issue by shark0der** [NomicFoundation/hardhat#1215]
When running `npm install @nomiclabs/hardhat-truffle5` there's a big red warning:

![image](https://user-images.githubusercontent.com/528959/106522760-d691c600-64e8-11eb-9d56-9b33a4e8a583.png)

This has been fixed upstream trufflesuite/truffle/issues/3551.
It seems that the fix is [really simple](https://github.com/trufflesuite/truffle/pull/3553/commits/373c865b3849a68058267d6496b958e673b16e94).

**Comment by shark0der** [comment]
Github's dependabot warns us that it cannot update `highlight.js` to a non-vulnerable version.

The latest possible version that can be installed is 9.18.5 because of the following conflicting dependencies:
```
@nomiclabs/hardhat-truffle5@2.0.0 requires highlight.js@^9.15.8 via a transitive dependency on @truffle/debug-utils@4.2.9
@nomiclabs/hardhat-truffle5@2.0.0 requires highlight.js@^9.12.0 via a transitive dependency on @trufflesuite/chromafi@2.2.0
```

The earliest fixed version is 10.4.1.

Vulnerability details: https://github.com/advisories/GHSA-7wwv-vh3v-89cq

It's worth noting that the last time I performed memory dump while running a hardhat script there was a lot of memory used by highlight.js. I don't know why this library is used while running scripts, perhaps some bloat inherited from truffle. Hopefully upgrading truffle to a newer version could fix this :grimacing:.

**Comment by fvictorio** [comment]
This doesn't seem to happen anymore in the latest version of the plugin.
